### ISSUE TITLE
Create a project/run README files 

### ISSUE THREAD

**Issue by Wytamma** [smutch/mccoy#21]
I would be good if we can generate some README files for the project/runs i.e. info on what reference/template/data are used and instructions on how to run the workflow etc. Maybe we could also populate the readme with results?

**Comment by smutch** [comment]
Definitely. I think it might be good to generate a snakemake report with the results. But we can do the project-specific readme as well.
